README.md

```diff
@@ -1,32 +1,52 @@
+![](https://raw.githubusercontent.com/Osekai/osekai/main/global/img/cover/medals.jpg)
+# *Osekai*
+Hey there, welcome to the official Osekai repository! We're an open-source website specializing in apps and tools for the rhythm game "osu!"
+
+If you're interested in helping out with development, read on! Else, if you just want to use the website, [check it out here](https://osekai.net)!
+
+
 # Setup
 Clone the folder locally. Create a database using MySQL or MariaDB and import the template.sql. Put info into config.php.
 
 Make oauth app on osu! and put in data to config.php, along with api v1 credentials.
 
+## **Running**
+
+### Local Php Server
+
 Run `php -S localhost:8080` in the folder, then access at http://localhost:8080
 
+### Apache Setup
+
+An Apache/httpd setup is required to run Osekai completely. Without it, certain apps such as Teams, Tools, and more will not function, and pages such as the 404 page will not show up. If you are planning to work on osekai alot, using apache is recommended.
+
+**The easiest way to do this is to install XAMPP, and then change the directory in the httpd/apache config to point to your Osekai clone.**
+
+If you feel like installing it manually, though, start by setting up apache like you usually would, you can follow a plethora of online tutorials for this. After you're done, just make sure it's using php8.1 and is pointing to the Osekai folder.
+
+You can use [this script](https://pastebin.com/3XGT5HwE) I've made to quickly switch between folders, keep in mind this'll only work on Arch Linux, so you'll need to edit the script for other distros. - `sudo switch /home/tanza/Documents/orgs/osekai/osekai` 
 
-## Database Setup
+## **Database Setup**
 Create a new user, call it whatever, whatever password, just remember it.
 
-Make a new database called `osekai`, you can name it whatever, but osekai is recommended. It's recommended to make it using utf8mb4_polish_ci
+Make a new database called `osekai`, you can name it whatever, but osekai is recommended. It's recommended to make it using **ufg8mb4_unicode_c1**
```

Comment:
i think the name of this collate is wrong, this should be the correct one i think
```suggestion
Make a new database called `osekai`, you can name it whatever, but osekai is recommended. It's recommended to make it using **utf8mb4_unicode_ci**
```

Reply:
achh i misspelt utf thank you for pointing that out